recipes/mitoz/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "3.0" %}
+
+# see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html
+# for how to define a meta.yaml
+
+package:
+  name: mitoz
+  version: {{ version }}
+
+build:
+  skip: True # [osx]
+  noarch: generic
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - mitoz = mitoz.MitoZ:main
+    - mitoz-tools = mitoz.tools:main
+
+source:
+  url: https://github.com/linzhi2013/MitoZ/releases/download/{{ version }}/mitoz-{{ version }}.tar.gz
+  md5: 2b552279ad16ca328f0723305713855c
+
+requirements:
+  host:
+    - python=3.6.7
+    - pip
+
+  run:
+    - fastp
+    - requests
+    - bs4
+    - libgd
+    - biopython=1.76
+    - blast=2.9.0
+    - bwa=0.7.17
+    - circos=0.69
+    - ete3=3.1.1
+    - hmmer=3.1b2
+    - infernal=1.1.1
+    - openjdk
+    - perl=5.26
+    - perl-list-moreutils
+    - perl-params-validate
+    - perl-clone
+    - perl-bioperl
+    - python=3.6.7
+    - samtools=1.3.1
```

Comment:
It needs these exact versions of python and samtools?

Reply:
Maybe Python 3.6 and other versions of samtools are okay, but I did not test them. Should we leave them to solve by conda? (although I do not expect too many differences for later versions of samtools). thanks!